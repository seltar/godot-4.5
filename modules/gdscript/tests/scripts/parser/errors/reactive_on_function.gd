extends RefCounted

# This should cause a parser error - @reactive on function
@reactive
func invalid_function():
	pass

func test():
	print("This should not run due to parser error")
