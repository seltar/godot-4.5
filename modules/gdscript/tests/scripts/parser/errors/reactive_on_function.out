GDTEST_PARSER_ERROR
Annotation "@reactive" cannot be applied to a function.
